docs/document/docusaurus.config.js

```diff
@@ -0,0 +1,163 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/okaidia');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
```

Comment:
i have fixed

Reply:
Confirmed the fix.

Reply:
How was the color theme switching requirement we discussed in the zoom chat?

Reply:
I have added dark mode in the new commit, please check

Reply:
Thank you!
On the component preview, some components are transparent and cannot see well.
For example, ReadOnlyTable, Table, Tabs.

If we can set white background color for all components, it will solve the problem, but if it seems difficult, I thought we should skip dark mode.

Reply:
I have rollback, and now we are only use light mode

Reply:
Got it. Thanks for the hard work on the investigation!